include/unknown_structs.h

```diff
@@ -5,6 +5,13 @@
 #include "libc/stdint.h"
 #include "libc/stdbool.h"
 #include "unk.h"
+#include "other_types.h"
 
+typedef struct D_8010EAA0 {
+    /* 0x00 */ RomOffset unk_00;
+    /* 0x04 */ RomOffset unk_04;
+    /* 0x08 */ char unk08[0xB];
+    /* 0x13 */ u8 unk_13;
+} D_8010EAA0;  // size = 0x14
```

Comment:
Could you please not name structs like if they were symbols?
You know our naming conventions for this kind of stuff
```suggestion
typedef struct struct_8010EAA0 {
    /* 0x00 */ RomOffset unk_00;
    /* 0x04 */ RomOffset unk_04;
    /* 0x08 */ char unk08[0xB];
    /* 0x13 */ u8 unk_13;
} struct_8010EAA0;  // size = 0x14
```